templates/repo/header.tmpl

```diff
@@ -18,9 +18,7 @@
                 </a>
                 <div id="repo-header-download-drop" class="drop-down">
                     <div id="repo-clone" class="clear">
-                        {{if not $.DisableSSH}}
-                        <button class="btn btn-blue left btn-left-radius" id="repo-clone-ssh" data-link="{{$.CloneLink.SSH}}">SSH</button>
-                        {{end}}
+                        <button class="btn btn-blue left btn-left-radius" id="repo-clone-ssh" {{if $.DisableSSH}}style="visibility: hidden" {{end}} data-link="{{$.CloneLink.SSH}}">SSH</button>
```

Comment:
Because there are layout issues on Firefox / narrower browser width.
I do not want to change the left button's css (relates to layout issues),
so I just hide the button